packages/big-design/src/components/Timepicker/Timepicker.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef, memo, Ref } from 'react';
+
+import { Select } from '../Select';
+
+interface Props {
+  error?: React.ReactNode;
+  label?: string;
+  onTimeChange(date: string): void;
+}
+
+interface PrivateProps {
+  forwardedRef: Ref<HTMLInputElement>;
+}
+
+export type TimepickerProps = Props & React.InputHTMLAttributes<HTMLInputElement>;
+
+function getTimeIntervals() {
```

Comment:
Other countries use 24hr time, what are your thoughts on making this (and I say this loosely) "locale aware"? We will down the line use some sort of locale service that gives us how to display customer information (date, time, currency, ltr/rtl, etc...), so how can we give it the option to use 24hr vs. 12hr times?

Reply:
@chanceaclark brings a good point, we should consider this.

Reply:
I did look at it quickly, based on locales and on using `https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat`. We get time back in AM/PM format by default unless we specify `hour12: false` in options. And that doesn't seem to be locale dependent.

Reply:
>And that doesn't seem to be locale dependent.

It is locale dependent. Take a look at which countries use 24hr vs. 12hr. https://en.wikipedia.org/wiki/Date_and_time_representation_by_country

Using the `.format` with France as the country will output it in 24hr time:
```javascript
let d = new Date("2019-01-01T00:00:00.000000Z");
let formattedDate = Intl.DateTimeFormat('fr', {
  year: 'numeric',
  month: 'numeric',
  day: 'numeric',
  hour: 'numeric',
  minute: 'numeric',
  second: 'numeric'
}).format(d);

formattedDate // "31/12/2018 à 18:00:00"
```

Reply:
I think I didn't explain my concern well, I did take a look at the above. Let me try and figure out a way to get the time values as per locale.